src/Uno.UI.Wasm/ts/Windows/Graphics/Display/DisplayInformation.ts

```diff
@@ -0,0 +1,56 @@
+﻿namespace Windows.Graphics.Display {
+
+	export class DisplayInformation {
+		private static readonly DpiCheckInterval = 500;
+
+		private static lastDpi: number;
+		private static dpiWatcher: number;
+
+		private static dispatchOrientationChanged: (type: string) => number;
+		private static dispatchDpiChanged: (dpi: number) => number;
+
+		public static startOrientationChanged() {
+			window.screen.orientation.addEventListener("change", DisplayInformation.onOrientationChange);
+		}
+
+		public static stopOrientationChanged() {
+			window.screen.orientation.removeEventListener("change", DisplayInformation.onOrientationChange);
+		}
+
+		public static startDpiChanged() {
+			// DPI can be observed using matchMedia query, but only for certain breakpoints
+			// for accurate observation, we use polling
+
+			DisplayInformation.lastDpi = window.devicePixelRatio;
+
+			// start polling the devicePixel
+			DisplayInformation.dpiWatcher = window.setInterval(DisplayInformation.updateDpi, DisplayInformation.DpiCheckInterval);
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5885845)